src/resolvers/resolver.ts

```diff
@@ -35,6 +54,53 @@ const resolvers = {
       await userExists.update({ profile: newProfile._id });
       return newProfile;
     },
+
+    async createCohort(_: any, { name, coordinator, phase }: any) {
+      const cohortExists = await Cohort.findOne({ name });
+
+      if (cohortExists) {
+        throw new Error('This cohort already exists ');
+      }
+      const newCohort = await Cohort.create({
+        name,
+        coordinator,
+        phase,
+      });
+      return newCohort;
+    },
+
+    async addMemberToCohort(_: any, { cohortName, memberId }: any) {
+      const isMember = await Cohort.findOne({ members: memberId });
+      const cohortExists = await Cohort.findOne({ name: cohortName });
+
+      if (isMember) {
+        throw new Error('This member is already added this cohort');
+      } else if (!cohortExists) {
+        throw new Error('This cohort does not exist ');
+      } else {
+        cohortExists.members.push(memberId);
+        cohortExists.save();
```

Comment:
This function is asynchronous and have to be called with an `await`

Reply:
> This function is asynchronous and have to be called with an `await`

true, it's been adressed.